pkg/services/ngalert/metrics/state.go

```diff
@@ -17,5 +19,24 @@ func NewStateMetrics(r prometheus.Registerer) *State {
 			Name:      "alerts",
 			Help:      "How many alerts by state.",
 		}, []string{"state"}),
+		StateUpdateDuration: promauto.With(r).NewHistogramVec(
+			prometheus.HistogramOpts{
+				Namespace: Namespace,
+				Subsystem: Subsystem,
+				Name:      "state_calculation_duration_milliseconds",
```

Comment:
Should we make this seconds just so its consistent with our other metrics? I don't see an issue using `0.01` etc for `10`?